packages/tx/README.md

```diff
@@ -112,15 +134,15 @@ const txData = {
   chainId: '0x01',
   accessList: [],
   type: '0x05',
-  versionedHashes: ['0xabc...'],
-  kzgCommitments: ['0xdef...'],
-  blobs: ['0xghi...'],
+  versionedHashes: ['0xabc...'], // Test with empty array on a first run
+  kzgCommitments: ['0xdef...'], // Test with empty array on a first run
+  blobs: ['0xghi...'], // Test with empty array on a first run
```

Comment:
Could we elaborate a bit more on why someone would "test with empty array on a first run"? (Perhaps in the instructions)

Reply:
This actually has no magical subcontext, but is rather to see if things work in general, which is easier to test/split without the actual data (then one just see if the KGZ setup works fine, the block constructs without errors,...).